### PR TITLE
feat(icons): add chart-cube-root and chart-lines-crossed icons

### PR DESCRIPTION
**Pull request by qshaner**
## Summary

Adds `Chart-cube-root` 
Screenshot 2024-10-01 at 8 45 09 AM

and `chart-crossing-lines`
Screenshot 2024-10-01 at 8 45 28 AM

icons for use in Flexible Stats

### CODE REVIEW

File: .changeset/odd-poems-divide.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@launchpad-ui/icons": minor
```
Comment (Niznikr) [inline .changeset/odd-poems-divide.md:2]:
```suggestion
"@launchpad-ui/icons": patch
```
`0.x` versioning 😄

Comment (qshaner) [reply .changeset/odd-poems-divide.md:2]: Ope, thank you!

**Comment by matthewferry** [comment]
Looks like the stroke width is a little off (should be 1.5px at 20x20 but looks thinner). Who is the designer on this? I'll loop up with them and reply back with an updated svg path 😄

**Review by matthewferry — CHANGES_REQUESTED** [review]
Needs updated path for 1.5px stroke

**Review by awinders-launchdarkly — APPROVED** [review]

File: packages/icons/src/img/sprite.svg [hunk 2]
```diff
@@ -451,6 +451,26 @@
...
+      <path 
+      fill-rule="evenodd" 
+      clip-rule="evenodd" 
+      d="M1.3999 12.6667C1.3999 12.998 1.66853 13.2667 1.9999 13.2667C5.32907 13.2667 6.99619 10.6721 8.44076 8.42381C8.46209 8.39061 8.48337 8.35749 8.50461 8.32446C10.0451 5.92821 11.3782 3.93334 13.9999 3.93334C14.3313 3.93334 14.5999 3.66471 14.5999 3.33334C14.5999 3.00197 14.3313 2.73334 13.9999 2.73334C10.6707 2.73334 9.00362 5.32795 7.55904 7.5762C7.53771 7.60939 7.51643 7.64251 7.4952 7.67555C5.95475 10.0718 4.62158 12.0667 1.9999 12.0667C1.66853 12.0667 1.3999 12.3353 1.3999 12.6667Z"/>
```
Comment (matthewferry) [inline packages/icons/src/img/sprite.svg:462]:
```suggestion
      <path 
        fill-rule="evenodd"
        d="M.75 10a.75.75 0 0 1 .75-.75h1a.75.75 0 0 1 0 1.5h-1A.75.75 0 0 1 .75 10Zm4 0a.75.75 0 0 1 .75-.75h1a.75.75 0 0 1 0 1.5h-1a.75.75 0 0 1-.75-.75Zm8 0a.75.75 0 0 1 .75-.75h1a.75.75 0 0 1 0 1.5h-1a.75.75 0 0 1-.75-.75Zm4 0a.75.75 0 0 1 .75-.75h1a.75.75 0 0 1 0 1.5h-1a.75.75 0 0 1-.75-.75Zm-9.49 5.102c-1.274 1.33-2.811 2.315-4.76 2.315a.75.75 0 0 1 0-1.5c1.385 0 2.556-.683 3.677-1.853 1.134-1.183 2.134-2.777 3.187-4.462l.025-.04c1.024-1.637 2.1-3.358 3.35-4.664 1.275-1.33 2.812-2.315 4.761-2.315a.75.75 0 0 1 0 1.5c-1.384 0-2.556.683-3.677 1.853-1.134 1.183-2.135 2.777-3.187 4.461l-.025.04c-1.024 1.638-2.1 3.36-3.35 4.665Z"
        clip-rule="evenodd"
      />
```

File: packages/icons/src/img/sprite.svg [hunk 3]
```diff
@@ -451,6 +451,26 @@
...
+    </symbol>
+    <symbol  viewBox="0 0 20 20" id="lp-icon-chart-crossing-lines">
+      <path 
+      fill-rule="evenodd" 
+      clip-rule="evenodd" 
+      d="M1.80005 4.00001C1.80005 3.66864 2.06868 3.40001 2.40005 3.40001C5.66047 3.40001 7.97791 4.45543 9.79557 6.04883C11.5737 7.60762 12.8578 9.67138 14.0854 11.6444L14.1095 11.683C14.2845 11.9644 14.1984 12.3344 13.917 12.5094C13.6357 12.6845 13.2657 12.5983 13.0906 12.317C11.8366 10.3017 10.637 8.38223 9.00453 6.9512C7.39997 5.5446 5.36185 4.60001 2.40005 4.60001C2.06868 4.60001 1.80005 4.33139 1.80005 4.00001Z"/>
+      <path 
+      fill-rule="evenodd" 
+      clip-rule="evenodd" 
+      d="M1.80005 12C1.80005 12.3314 2.06868 12.6 2.40005 12.6C5.66047 12.6 7.97791 11.5446 9.79557 9.9512C11.5737 8.39241 12.8578 6.32865 14.0854 4.35563L14.1095 4.317C14.2845 4.03564 14.1984 3.66564 13.917 3.49058C13.6357 3.31552 13.2657 3.40168 13.0906 3.68303C11.8366 5.69835 10.637 7.6178 9.00453 9.04883C7.39997 10.4554 5.36185 11.4 2.40005 11.4C2.06868 11.4 1.80005 11.6686 1.80005 12Z"/>
```
Comment (matthewferry) [inline packages/icons/src/img/sprite.svg:472]:
```suggestion
      <path
        fill-rule="evenodd"
        d="M1.25 5A.75.75 0 0 1 2 4.25c4.838 0 8.018 1.296 10.377 3.304 2.138 1.82 3.556 4.196 4.924 6.488l.342.572a.75.75 0 1 1-1.286.772c-.108-.18-.215-.358-.32-.536-1.396-2.334-2.691-4.502-4.632-6.154C9.357 6.954 6.537 5.75 2 5.75A.75.75 0 0 1 1.25 5Z"
        clip-rule="evenodd"
      />
      <path
        fill-rule="evenodd"
        d="M1.25 15c0 .414.336.75.75.75 4.838 0 8.018-1.296 10.377-3.304 2.138-1.82 3.556-4.196 4.924-6.488l.342-.572a.75.75 0 1 0-1.286-.772c-.108.18-.215.358-.32.536-1.396 2.334-2.691 4.502-4.632 6.154C9.357 13.046 6.537 14.25 2 14.25a.75.75 0 0 0-.75.75Z"
        clip-rule="evenodd"
      />
```

File: packages/icons/src/types.ts [hunk 4]
```diff
@@ -60,6 +60,8 @@ const icons = [
 	'change-history',
 	'chart-area',
 	'chart-bar',
+	'chart-cube-root',
+	'chart-crossing-lines',
```
Comment (matthewferry) [inline packages/icons/src/types.ts:64]: nit: but maybe call this `chart-lines-crossing` to align with our other `chart-line-*` icons

Comment (qshaner) [reply packages/icons/src/types.ts:64]: that makes a lot more sense, thank you!

**Review by Niznikr — APPROVED** [review]

**Review by matthewferry — APPROVED** [review]

**Pull request merged**